lib/puppetserver/ca/local_certificate_authority.rb

```diff
@@ -36,11 +37,37 @@ class LocalCertificateAuthority
         ["authorityKeyIdentifier", "keyid:always", false]
       ].freeze
 
+      attr_reader :cert, :key, :crl
+
       def initialize(digest, settings)
         @digest = digest
         @host = Host.new(digest)
         @settings = settings
         @errors = []
+
+        if ssl_assets_exist?
+          @cert, @key, @crl = load_ssl_assets
+        end
+      end
+
+      def ssl_assets_exist?
+        File.exist?(@settings[:cacert]) &&
+          File.exist?(@settings[:cakey]) &&
+          File.exist?(@settings[:cacrl])
+      end
+
+      def load_ssl_assets(settings=@settings)
+        loader = Puppetserver::Ca::X509Loader.new(@settings[:cacert], @settings[:cakey], @settings[:cacrl])
+        if loader.errors.empty?
+          cert = OpenSSL::X509::Certificate.new(File.read(settings[:cacert]))
+          key = OpenSSL::PKey::RSA.new(File.read(settings[:cakey]))
+          crl = OpenSSL::X509::CRL.new(File.read(settings[:cacrl]))
```

Comment:
The loader should already save these in instance variables when it's done loading them, can we just use those so we don't have to read the files twice? (Could get expensive if the CRL is large.)